Lib/test/cov.py

```diff
@@ -0,0 +1,22 @@
+"""A minimal hook for gathering line coverage of the standard library."""
+
+import sys
+
+mon = sys.monitoring
+mon.use_tool_id(mon.COVERAGE_ID, "regrtest coverage")
+
+FileName = str
+LineNo = int
+Location = tuple[FileName, LineNo]
+COVERAGE: set[Location] = set()
```

Comment:
Usually, uppercase is for constants. Maybe rename to lowercase?

Would it make sense to make it private (add an underscore) and add a getter function? For example, currently worker_process() creates a copy by calling `list(COVERAGE)`.